### ISSUE TITLE
Tracking data about physical storage locations

### ISSUE THREAD

**Issue by ekrimmel** [paleo-data/happy-hours#34]
## What is your idea for a happy hour topic?
Physical storage locations, including organizational schemes, digital location management in CMS, barcoding, wayfinding, etc.

## Who could be involved in leading this happy hour?
Suzanne Mills

## Are there any timing considerations?
_[add text here - e.g. needs to happen before xyz conference, after field season, ASAP, etc.]_

## Next steps
This issue captures a happy hour topic idea. Once you are ready to finalize an idea and put it on the PDWG happy hour schedule, please create a new issue using the [Plan a happy hour](https://github.com/paleo-data/happy-hours/issues/new?assignees=&labels=PDWG&projects=&template=plan-happy-hour.md&title=%5Breplace+this+text+with+the+happy+hour+topic%5D) template. **Then, close this issue.**

**Comment by ekrimmel** [comment]
From discussion at HH 2025-01-16
- this is an institutional knowledge issue (Suzanne) +1 (Amanda)
- could tie in well with the topic about #31 
- could also tie in with #22
